healthScore/healthConfig/settings.py

```diff
@@ -82,7 +82,7 @@
         'ENGINE': 'django.db.backends.mysql',
         'NAME': 'HealthScore',
         'USER': 'root',
-        'PASSWORD': '', # change it with your sql server password for local testing
+        'PASSWORD': 'root123456789', # change it with your sql server password for local testing
```

Comment:
You can remove this password and leave it as an empty string itself

Reply:
Fixed and pushed again to cindy_dev